modules/ossm-installing-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+
+// * service-mesh-docs-main/install/ossm-istio-ambient-mode.adoc
+
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-istio-ambient-mode_{context}"]
+= Installing Istio ambient mode
+
+You can install {istio} ambient mode on {ocp-product-title} 4.19 or later and {SMProductName} 3.1.0 or later with the required Gateway API custom resource definitions (CRDs).
+
+.Prerequisites
+
+* You have deployed a cluster on {ocp-product-title} 4.19 or later, which includes the supported {k8s} Gateway API CRDs required for {istio} ambient mode.
+* You have installed {SMProduct} Operator 3.1.0 or later in the {ocp-product-title} cluster.
+* You are logged in to the {ocp-product-title} either through the web console as a user with the `cluster-admin` role, or with the `oc login` command, depending on the installation method.
+
+.Procedure
+
+. Install the {istio} control plane:
+
+.. Create the `istio-system` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc create namespace istio-system
+----
+
+.. Create the `{istio}` resource similar to the following example:
++
+.Example configuration
+[source, yaml]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  profile: ambient
+  values:
+    pilot:
+      trustedZtunnelNamespace: ztunnel
+----
++
+[NOTE]
+====
+The `profile` field should be set to `ambient`, and configure the `.spec.values.pilot.trustedZtunnelNamespace` value to match the namespace where the `ZTunnel` resource will be installed.
+====
+
+.. Apply the `{istio}` custom resource (CR) by running the following command:
++
+[source, terminal]
+----
+$ oc apply -f istio.yaml
+----
+
+.. Wait for the {istio} control plane to contain the `Ready` status condition by running the following command:
++
+[source, terminal]
+----
+$ oc wait --for=condition=Ready istios/default --timeout=3m
+----
+
+. Install the {istio} Container Network Interface (CNI):
+
+.. Create the `istio-cni` namespace by running the following command:
++
+[source, terminal]
+----
+$ oc create namespace istio-cni
+----
+
+.. Create the `IstioCNI` resource similar to the following example:
```

Comment:
Same suggestion here about making it more clear that the file created in this step is the one being applied in the next one